extensions.json

```
@@ -1638,8 +1632,7 @@
     {
       "id": "vscodevim.vim",
       "download": "https://github.com/VSCodeVim/Vim/releases/download/v1.21.3/vim-1.21.3.vsix",
-      "version": "1.21.3",
-      "web": true
+      "version": "1.21.3"
```

Wow, this looks new. No idea what it does, but we probably shouldn't remove it:

```suggestion
      "version": "1.21.3",
      "web": true
```